CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v2.7.1
+
+*07 sep 2022*
+
+- Added support for smart read pixels after draw (#417)
```

Comment:
This isn't entirely accurate as there are other important hotfixes that were merged directly to master already that haven't been released:
- #414 
- #413 
- #415

Reply:
Thanks Frank.

@sandeep-vedam can you add these other hotfixes to the PR and change the date to coming, then we'll roll out the release then. Thx!